testing/test_physics_metrics.py

```diff
@@ -286,11 +306,21 @@ def postProcessing(data, preds):
     # set all particle masses to zero
     mass = np.zeros(pt.shape[0])
 
+    testneu = 1
```

Comment:
here it is probably fine to hard code for now. but in general this is a not a good habit..